location/utils.py

```diff
@@ -99,3 +99,81 @@ def compute_total_gyration(data, sr_col='stay_region'):
         temp_sum += loc_cnt[loc] * vincenty(geohash.decode(loc), r_cm).m ** 2
 
     return math.sqrt(1/N * temp_sum)
+
+
+def compute_gyration(data,
+                     sr_col='stay_region',
+                     k=None,
+                     remove_dist_pt=True,
+                     home=None,
+                     dist_pt_th=50000):
+    """
+    Compute radius of gyration.
```

Comment:
Not exactly sure what is the difference between this function and `compute_total_gyration`. The document should clarify that.